app/views/system_admin/_tab_nav.html.erb

```diff
@@ -5,7 +5,8 @@
   { name: "Schools", url: schools_path },
   { name: "Lead schools", url: lead_schools_path },
   { name: "Uploads", url: uploads_path },
-  { name: "Dead Jobs", url: dead_jobs_path },
+  { name: "Dead jobs", url: dead_jobs_path },
   { name: "Pending TRN", url: pending_trns_path },
-  { name: "Pending Awards", url: pending_awards_path },
+  { name: "Pending awards", url: pending_awards_path },
+  { name: "Duplicate apply applications", url: duplicate_apply_applications_path },
```

Comment:
It feels like it should be `Apply` rather then `apply` as it is referring to our sister service